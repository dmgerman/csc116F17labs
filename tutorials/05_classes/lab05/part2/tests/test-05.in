10:asdf
this_is:invalid