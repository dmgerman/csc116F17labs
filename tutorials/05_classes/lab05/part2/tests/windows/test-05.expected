Enter a time: 
You entered "10:asdf"
Runtime error: Invalid time
